framework/test/bundles/libU/resources/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "bundle.symbolic_name" : "TestBundleU",
+  "bundle.activator" : true,
+  "unicode.sample" : "电脑 くいりのまちとこしくそ"
```

Comment:
The property does not seem to be used. It would be nice to extend the unit tests and compare the manifest header with an embedded `u8"..."` string literal.